tox.ini

```diff
@@ -45,7 +45,7 @@ conda_deps =
 conda_channels = sunpy
 commands =
     offline,astropydev,numpydev: pytest -m "not figure" --cov=./sunpy {posargs}
-    online: pytest -m "not figure" --remote-data=any --cov=./sunpy {posargs}
+    online: pytest -vvv -m "not figure" --remote-data=any --cov=./sunpy {posargs}
```

Comment:
revert?

Reply:
I will once i see what is failing on travis but not on azure online test.